docs/migration-7.10-11.md

```diff
@@ -0,0 +1,168 @@
+# ICM 7.10 to ICM 11 Migration Steps
+
+This document outlines the migration process from ICM 7.10 to ICM 11, including automated steps performed by the
+migration tool and manual steps required afterward.
+
+## Table of Contents
+
+- [Automated Migration Steps](#automated-migration-steps)
+    - [Remove Assembly Projects](#remove-assembly-projects)
+    - [Move Folder Structure](#move-folder-structure)
+    - [Convert build.gradle Files](#convert-buildgradle-files)
+    - [Convert to Cartridge Dependency](#convert-to-cartridge-dependency)
+    - [Rename Dependencies](#rename-dependencies)
+    - [Remove Obsolete Dependencies](#remove-obsolete-dependencies)
+    - [Move DB Prepare Files](#move-db-prepare-files)
+    - [Migrate Configuration Resources](#migrate-configuration-resources)
+    - [Migrate Version Information](#migrate-version-information)
+    - [Add Site Content Preparer](#add-site-content-preparer)
+    - [Rename Packages](#rename-packages)
+    - [Delete Obsolete Files](#delete-obsolete-files)
+- [Manual Migration Steps](#manual-migration-steps)
+    - [Global defined dependencies](#global-defined-dependencies)
+    - [Remove Sites Folder Copy Tasks](#remove-sites-folder-copy-tasks)
+
+## Automated Migration Steps
+
+### Remove Assembly Projects
+
+Migrator: `RemoveAssembly`
+
+- Assembly projects handled cartridge lists for deployment in ICM 7.10
+- Removes assembly projects which are no longer needed in ICM 11
+
+### Move Folder Structure
+
+Migrator: `MoveFolder`
+
+Moves all staticfiles to their new locations in the ICM 11 structure:
+
+- `staticfiles/cartridge/pipelines` -> `src/main/resources/resources/{cartridgeName}/pipelines`
```

Comment:
I'd suggest to add a small note that the `{cartridgeName}` is a placeholder and replaced by the current cartridge migrated.